service/policy/obligations/obligations.proto

```diff
@@ -0,0 +1,230 @@
+syntax = "proto3";
+
+package policy.obligations;
+
+import "common/common.proto";
+import "policy/objects.proto";
+import "policy/selectors.proto";
+import "google/protobuf/struct.proto";
+
+///
+/// Obligation
+///
+
+// Definitions
+message GetObligationRequest {
+    string id = 1;
+}
+
+message GetObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message CreateObligationRequest {
+    // Required
+    string namespace_id = 1;
+    string name = 2;
+    // Optional
+    // Common metadata
+    common.MetadataMutable metadata = 100;
+}
+
+message CreateObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message UpdateObligationRequest {
+    // Required
+    string id = 1;
+
+    // Optional
+    common.MetadataMutable metadata = 100;
+    common.MetadataUpdateEnum metadata_update_behavior = 101;
+}
+
+message UpdateObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message DeleteObligationRequest {
+    string id = 1;
```

Comment:
Should this be oneof `id` or `fqn`?

Reply:
I think `id` is preferred unless it's unsafe, and deleting an obligation does not retroactively affect access to an existing TDF (just PEP capabilities & behaviors), so it's safe in my opinion.